### ISSUE TITLE
Element not highlighted on focus H2

### ISSUE THREAD

**Issue by paazmaya** [paazmaya/espoo.kobujutsu.fi#27]
With keyboard navigation it is important that the focused component can be visually identified. If the focused component is not somehow highlighted, the user cannot be sure which component has focus.
